src/main/java/net/playeranalytics/extension/quests/QuestsExtension.java

```diff
@@ -73,23 +76,16 @@ public CallEvents[] callExtensionMethodsOn() {
         };
     }
 
-    @StringProvider(
+    @NumberProvider(
             text = "Quest Points",
             description = "Total amount of Quest Points",
             priority = 100,
             iconName = "certificate",
             iconColor = Color.AMBER
     )
     @Tab("Statistics")
-    public String points(UUID playerUUID) {
-        try {
-            return Optional.of(String.valueOf(getQuester(playerUUID).getQuestPoints())).orElse("0");
-        } catch (InterruptedException e) {
-            Thread.currentThread().interrupt();
-            throw new NotReadyException();
-        } catch (ExecutionException e) {
-            throw new NotReadyException();
-        }
+    public int points(UUID playerUUID) {
+        return Optional.of(getQuester(playerUUID).getQuestPoints()).orElse(0);
```

Comment:
if getQuestPoints returns `null` `of` will throw a NullPointerException - If it never returns null the use of optional is unnecessary